bin/fluent-ca-generate

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env ruby
+
+$LOAD_PATH.unshift(File.join(__dir__, 'lib'))
+require 'fileutils'
```

Comment:
This `require` should be in `lib/fluent/command/ca_generate.rb`?

Reply:
Oops.